src/wkwebview/mod.rs

```diff
@@ -370,6 +370,24 @@ impl InnerWebView {
         let frame = ns_view.frame();
         let webview: Retained<WryWebView> =
           objc2::msg_send![super(webview), initWithFrame: frame, configuration: &**config];
+        if let Some((red, green, blue, alpha)) = attributes.background_color {
+          // This is required first since the webview color is applied too late.
+          webview.setOpaque(false);
+
+          let color = objc2_ui_kit::UIColor::colorWithRed_green_blue_alpha(
+            (red / 255).into(),
```

Comment:
I think we should convert the value into a floating point like `(red as f64 / 255.0)` otherwise the value < 255 will present as 0.

Reply:
idk how i missed that, did i really only test colors that are close to 0 or 255 in all 3 colors? I mean, i'm colorblind but that's a bit wild 😂

Reply:
yes